src/TimesheetGPT.Core/Plugins.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.SemanticKernel;
+using System.ComponentModel;
+using System.Globalization;
+using TimesheetGPT.Core.Interfaces;
+using System.Text.Json;
+
+
+namespace TimesheetGPT.Core;
+
+public class GraphPlugins(IGraphService graphService)
+{
+    [SKFunction, Description("Get email body from Id")]
```

Comment:
I like the attribute format from SK. Pretty sweet.

Can we pass a CancellationToken or no?

Reply:
You probably could get it to work but it would mean adding stuff to the prompt saying "here is the ct"